appsec-kit-backend/src/main/java/com/vaadin/appsec/backend/util/OpenSourceVulnerabilityUtil.java

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
I'm going to move this utils class to the V8 demo application as its functions are only used there in the DemoUI and are not needed for dependency dialog in this format.